test/upgrade/check-from-v0.7.1-schema-registry.td

```diff
@@ -0,0 +1,17 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+> SELECT * FROM data
+1
+
+# This next query will fail if a bug like #10234 occurs again, where a migration
+# injects a `WITH` option into the wrong spot.
+
+> SHOW CREATE SOURCE data
+"materialize.public.data" "CREATE SOURCE \"materialize\".\"public\".\"data\" FROM KAFKA BROKER 'kafka:9092' TOPIC 'testdrive-data-${testdrive.seed}' FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://schema-registry:8081/' SEED VALUE SCHEMA '\"int\"'"
```

Comment:
does this actually print out the wrong with value without my change?

Reply:
Yup!